everything.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I'm reading between the lines and believe this script is designed to help with the commit / test process.

Overall I like the idea, although not sure if it fits with this PR. Also I would argue for a clearer name than "everything"

Reply:
This was actually just something I used personally and was accidently committed. I can create a new PR for this later if you like the idea :)